### PR TITLE
Default to new CopyStaticAssetsPlugin, and provide flag to revert back to old plugin

### PR DESCRIPTION
**Pull request by calebmshafer**
@aruniverse let me know what you think of the flag and description. I was debating turning it on by default instead of requiring a flag. That way any app that's not using workspaces can get the advantages and we can just explicitly turn it off in the other repos...

Once we finish this PR, I'll port to the 4.x PR.

### CODE REVIEW

File: packages/react-scripts/config/webpack.config.js [hunk 1]
```diff
@@ -698,7 +700,9 @@ module.exports = function(webpackEnv) {
       // NOTE: iModel.js specific plugin to copy a set of static resources from the node_modules
       // directory of each dependent package into the 'build/public' directory.
       // Used for resources such as locales, which are defined by each consuming package.
-      new CopyStaticAssetsPlugin({}),
+      useNewAssetCopy
+        ? new CopyStaticAssetsPlugin({})
+        : new CopyBentleyStaticResourcesPlugin(['public'], true),
```
Comment (aruniverse) [inline packages/react-scripts/config/webpack.config.js:705]: You'll need to add the import for this

**Review by aruniverse — COMMENTED** [review]
This is a good alternative, could you bump the pkg ver too?

**Review by aruniverse — APPROVED** [review]

**Comment by aruniverse** [comment]
Actually thinking about it now, it's probably more preferable to default to the new plugin, and use a flag to use the old one when you're using workspaces.
Just so when users bootstrap a new app, strings will be localized and look a bit more professional right off the bat.

**Comment by calebmshafer** [comment]
> Actually thinking about it now, it's probably more preferable to default to the new plugin, and use a flag to use the old one when you're using workspaces.
> Just so when users bootstrap a new app, strings will be localized and look a bit more professional right off the bat.

@aruniverse agreed, I'll flip it around.

File: packages/react-scripts/config/webpack.config.js (hunk 1 above)
Comment (calebmshafer) [reply packages/react-scripts/config/webpack.config.js:706]: done.

**Review by wgoehrig — APPROVED** [review]

File: packages/react-scripts/README-imodeljs.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Current upstream with `react-scripts@3.4.1`.
...
   | DISABLE_TERSER          | 🚫 Ignored  | ✅ Used    | When set to `true`, skips all minification. Useful for PR builds and test apps.                                                                                                                                                                                                                                       |
-  | USE_NEW_ASSET_COPY      | ✅ Used     | ✅ Used    | When set to `true`, uses the new copy static assets workflow which copies all files from dependencies starting with `@bentley` or `@itwin` scope. In addition it would make them available to both a `build` and `start`. Note: There is a bug with the update when used in conjunction with pnpm or yarn workspaces. |
+  | DISABLE_NEW_ASSET_COPY  | ✅ Used     | ✅ Used    | When set to `true`, disables the new copy static assets workflow which copies all files from dependencies starting with `@bentley` or `@itwin` scope. In addition it makes them available to both a `build` and `start` scripts. Note: There is a bug with the new asset copy when used with pnpm or yarn workspaces. |
```
Comment (aruniverse) [inline packages/react-scripts/README-imodeljs.md:27]: nit
```suggestion
  | DISABLE_NEW_ASSET_COPY  | ✅ Used     | ✅ Used    | When set to `true`, disables the new copy static assets plugin which copies all files from dependencies starting with `@bentley` or `@itwin` scope. In addition it makes them available to both a `build` and `start` scripts. Note: There is a bug with the new asset copy when used with pnpm or yarn workspaces. |
```

File: packages/react-scripts/config/webpack.config.js [hunk 3]
```diff
@@ -701,7 +701,7 @@ module.exports = function(webpackEnv) {
       // NOTE: iModel.js specific plugin to copy a set of static resources from the node_modules
       // directory of each dependent package into the 'build/public' directory.
       // Used for resources such as locales, which are defined by each consuming package.
-      useNewAssetCopy
+      disableNewAssetCopy
```
Comment (aruniverse) [inline packages/react-scripts/config/webpack.config.js:704]:
```suggestion
      !disableNewAssetCopy
```

Comment (calebmshafer) [reply packages/react-scripts/config/webpack.config.js:704]: I just switched the ordering of the options instead.

File: packages/react-scripts/README-imodeljs.md (hunk 2 above)
Comment (calebmshafer) [reply packages/react-scripts/README-imodeljs.md]: fixed, thanks.

**Pull request merged**